panels/info/cc-info-panel.c

```diff
@@ -1840,6 +1843,24 @@ cc_info_panel_init (CcInfoPanel *self)
       return;
     }
 
+  self->priv->hbox1 = WID ("hbox1");
+  self->priv->detail_vbox = WID ("detail_vbox");
+
+  if (gdk_screen_width () < 1024 || gdk_screen_height () < 600)
```

Comment:
I wonder if we can't reuse the "small screen" property of CcWindow instead of hardcoding a size here